audobject/core/api.py

```diff
@@ -27,6 +28,8 @@ def from_dict(
     Args:
         d: dictionary representing the object
         root: if dictionary was read from a file, set to source directory
+        auto_install: install missing packages
```

Comment:
Not sure if it is obvious to a user what a missing package might be when loading a YAML file.
Maybe:
```suggestion
        auto_install: install missing packages needed to create the object
```